### DISCUSSION TITLE
Connessione Ariston Clima ad HA

### DISCUSSION THREAD

**Discussion by Teorozzo** [indomus/forum#2407] [GENERALE - Discussioni generiche]
Qualcuno sa come connettere un condizionatore Ariston (con modulo wifi) ad home Assistant?
Nemmeno cercando online sono riuscito a trovare risposte

**Comment by KC-inDomus** [comment]
che clima è?

**Reply by Teorozzo** [reply]
Il modello è ariston kitprios50mud0.
Ho installato anche la chiavetta per il modulo Wi-Fi

**Reply by KC-inDomus** [reply]
non trovo niente. alle brutte: https://indomus.it/progetti/domotizzare-un-condizionatore-tradizionale-con-broadlink-e-home-assistant-v2/

**Reply by Teorozzo** [reply]
Si, infatti anche io non trovavo nulla, però tentar non nuoce quindi ho provato a chiedere.
Avevo comunque già pensato al Broadlink, appena arriverà utilizzerò quello.
Grazie comunque
